scene/gui/base_button.cpp

```diff
@@ -41,7 +41,9 @@ void BaseButton::_unpress_group() {
 	}
 
 	if (toggle_mode) {
```

Comment:
```suggestion
	if (toggle_mode && !button_group->is_allow_unpress()) {
```
No need to nest this.